components/service/sync-autofill/src/test/java/mozilla/components/service/sync/autofill/AutofillCreditCardsAddressesStorageTest.kt

```diff
@@ -35,7 +36,10 @@ class AutofillCreditCardsAddressesStorageTest {
     fun `addCreditCard`() = runBlocking {
         val creditCardFields = UpdatableCreditCardFields(
             billingName = "Jon Doe",
-            cardNumber = "4111111111111111",
+            // For the sake of these tests, it's not necessary that we
+            // use a real encryption key or real ciphertext.
+            cardNumberEnc = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
```

Comment:
I am wondering about the API changes around how to add a credit card. I assume we should be using the cypto library to generate `cardNumberEnc ` here. I think it would be nice to take the original `cardNumber` as an example and emulate the real API usage with the crypto library for this test.